src/test/java/seedu/address/logic/parser/EditStudentCommandParserTest.java

```diff
@@ -205,8 +206,8 @@ public void parse_resetTags_success() {
         Index targetIndex = INDEX_THIRD_PERSON;
         String userInput = targetIndex.getOneBased() + TAG_EMPTY;
 
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder().withTags().build();
-        EditCommand expectedCommand = new EditCommand(targetIndex, descriptor);
+        EditStudentCommand.EditStudentDescriptor descriptor = new EditStudentDescriptorBuilder().withTags().build();
```

Comment:
should just use EditStudentDescriptor directly instead of referring to EditStudentCommand